resources/js/Pages/AnnualSummary.vue

```diff
@@ -73,16 +73,44 @@
                 </template>
               </Popper>
               <button
-                v-else
-                class="py-1.5 w-full font-medium bg-white hover:bg-blumilk-25 border-b-4 border-transparent focus:outline-blumilk-500"
+                v-else-if="day.isWeekend"
+                class="py-1.5 w-full font-medium bg-white hover:bg-blumilk-25 border-b-4 border-transparent focus:outline-blumilk-500 hover:bg-transparent cursor-not-allowed"
```

Comment:
I think that we should remove `cursor-not-allowed`, because it looks bad. Just leave mouse point cursor, but don't highlight this field if user hovers on this. Also please remember, that we using TailwindCSS and it has some classes related to cursors - you don't need to write it in styles, just use as class - [link].(https://tailwindcss.com/docs/cursor)